cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -50,6 +50,15 @@ def test_infra_mapping_ui_assertions(appliance, v2v_providers, form_data_single_
      mapping_list.get_map_source_networks(mapping.name)[0])
     soft_assert(mapping.form_data['network'].values()[0]['mappings'][0]['target'][0].format() in
      mapping_list.get_map_target_networks(mapping.name)[0])
+
+    rhv_prov = v2v_providers[1]
```

Comment:
**Required** You should be using the v2v namedtuple here.

Reply:
updated this and elsewhere. :)